apps/coscrad-frontend/src/components/resources/playlists/episode-presenter.tsx

```diff
@@ -0,0 +1,11 @@
+import { IMultilingualText, IPlaylistEpisode } from '@coscrad/api-interfaces';
+import { Card } from '@mui/material';
+
+interface EpisodePresenterProps {
+    episode: IPlaylistEpisode;
+    name: IMultilingualText;
+}
+
+export const EpisodePresenter = ({ episode }: EpisodePresenterProps): JSX.Element => {
+    return <Card>{episode.name}</Card>;
```

Comment:
Better yet, isn't this multilingual text? You should use a multilingual text presenter if so. Talk with Geoff about how he handled this.